docs/framework/network-programming/how-to-list-directory-contents-with-ftp.md

```diff
@@ -73,4 +73,14 @@ Namespace Examples.System.Net
         End Sub
     End Module
 End Namespace
-```
\ No newline at end of file
+```
+
+If you need to list a specific directory, just add the directory to the end of URI you're using in `WebRequest.Create` method:
```

Comment:
nit: end of URI in -> end of **the** URI in **the**

Reply:
One more small thing.
`WebRequest.Create` -> replace with the following syntax to get a link to the API docs:
`<xref:System.Net.WebRequest.Create%2A>`

Reply:
As is. It’s to represent a *. You can find more info at 
https://github.com/dotnet/docs/blob/master/styleguide/template.md#links-to-apis

Reply:
Thank